tests/quantization/bnb/test_4bit.py

```diff
@@ -635,7 +661,9 @@ def test_serialization(self, quant_type="nf4", double_quant=True, safe_serializa
                     d1[k].quant_state.as_dict().values(),
                 ):
                     if isinstance(v0, torch.Tensor):
-                        self.assertTrue(torch.equal(v0, v1.to(v0.device)))
+                        # The asbmax will not be saved in the quant_state when using NF4 in CPU
```

Comment:
Typo (probably?), but I'm also not following what the comment means.
```suggestion
                        # The absmax will not be saved in the quant_state when using NF4 in CPU
```